source/GameAction.cpp

```diff
@@ -239,6 +247,8 @@ void GameAction::Save(DataWriter &out) const
 		out.Write("fail", name);
 	if(failCaller)
 		out.Write("fail");
+	if (playMusic)
+		out.Write("play", music);
```

Comment:
The style is to not have any space after the `if`.
```suggestion
	if(playMusic)
		out.Write("play", music);
```